stable/rocketchat/.helmignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
This is not the standard `.gitignore`. Let's wait on the outcome of https://github.com/kubernetes/charts/pull/485/files#r106305261

Looks like we prefer the "canonical" helmignore from now on but many charts still have just this. It probably makes more sense to sync them up in a separate PR across all charts.